scripts/amd/gemm/tune_gemm.py

```diff
@@ -185,14 +203,15 @@ def matmul_{configStr}(a, b, c, M, N, K, am, ak, bk, bn, cm, cn, warmup=False):
             waves_per_eu = {waves_per_eu},
             matrix_instr_nonkdim = {mfmaInstrSize},
             kpack = {kpack},
-            grid=(1,)
+            grid=(1,),
+            BIAS={use_bias},
```

Comment:
It's better to put kernel arguments before `grid=()`

Reply:
thanks. fixed